src/formatter.rs

```diff
@@ -31,10 +31,15 @@ impl Settings {
         let r = read_to_string(path_from)?;
         let s = self.format(&r);
         let mut file = File::create(path_to)?;
-        file.write_all(s.as_bytes())?;
-        return Ok(());
+        match s {
```

Comment:
Why? Why not do `self.format(&r)?;` at line 32?

Reply:
Hello @ice1000, 

I did not try it honestly. The signature is for `Result<(), std::io::Error>`. The error from format result is &str.

![image](https://user-images.githubusercontent.com/12170/91652707-9b2d3000-ea4e-11ea-9366-cf883337d1bd.png)

Thanks for your time in the review.

Dictated but not reviewed,
Becker

Reply:
I guess you can try `.map_err(|e| Error::new(ErrorKind::Other, format!("{}", e)))?`